cmake/CMakeLists.txt

```diff
@@ -90,6 +90,7 @@ option(onnxruntime_ENABLE_LANGUAGE_INTEROP_OPS "Enable operator implemented in l
 option(onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS "Dump node input shapes and output data to standard output when executing the model." OFF)
 option(onnxruntime_USE_DML "Build with DirectML support" OFF)
 option(onnxruntime_USE_WINML "Build with WinML support" OFF)
+option(onnxruntime_BUILD_WINML_IN_MICROSOFT_NS "Build the WinML APIs into the Microsoft namespace" OFF)
```

Comment:
Shouldn't this be on by default, and set to OFF only for inbox builds? If I'm an external user of the lib and want to build it myself shouldn't be in the Windows namespace, right?

Reply:
it is defaulted to on by the build.py.

Reply:
Do we want Windows to be the default though? One could run cmake directly or take an existing workspace and set onnxruntime_USE_WINML

Reply:
build.py isn't the only way one could build WinML

Reply:
Yeah, I thought of this, but figured that using winml from source will require a number of arguments to build correctly and so another wouldn't hurt.

In any case I added Microsoft as the default and added a new param.